etc/ci/scenario/common_function_for_servo_test.py

```diff
@@ -0,0 +1,64 @@
+import re
+import os
+import shutil
+import subprocess
+from decimal import Decimal
+from selenium import webdriver
+from selenium.webdriver.common.options import ArgOptions
+
+WEBDRIVER_PORT = 7000
+SERVO_URL = f"http://127.0.0.1:{WEBDRIVER_PORT}"
+
+def calculate_frame_rate():
+    """
+    Pull trace from device and calculate frame rate through trace
+    :return: frame rate
+    """
+    path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'target/ci_testing')
```

Comment:
`pathlib.Path(os.path.dirname(__file__)).parent.parent.parent.join('target/ci_testing')`

I think using `parent` on `Pathlib.Path` would be a bit clearer, then calling `dirname` multiple times.
Perhaps we should also split this into 2 steps: 
1. ``pathlib.Path(os.path.dirname(__file__)).parent.parent.parent.join('target)` and assert that `target` exists. If it does not, then probably we got the Path wrong.
2. now do `.join(ci_testing)`, check if that exists and create folder